examples/actions/src/main.rs

```diff
@@ -0,0 +1,158 @@
+#![allow(rustdoc::missing_crate_level_docs)]
+
+use eframe::{App, Frame, egui};
+use egui::{Align2, Color32, Context, RichText};
+use egui_plot::{Line, Plot, PlotEvent, TooltipOptions};
+
+fn main() -> eframe::Result<()> {
+    eframe::run_native(
+        "Plot + actions + pins demo",
+        eframe::NativeOptions::default(),
+        Box::new(|_| Ok(Box::new(Demo::new()))),
+    )
+}
+
+struct Demo {
+    xs: Vec<f64>,
+    f1: Vec<f64>,
+    f2: Vec<f64>,
+    last_event: Option<String>,
+}
+
+impl Demo {
+    fn new() -> Self {
+        let n = 500;
+        let xs: Vec<f64> = (0..n).map(|i| i as f64 * 0.02).collect();
```

Comment:
Pls change the code to be with `t` parametrization, `t` being in interval (0, 2*pi)
and small number of samples n=10 to show "rough edges" as they smoothen out

so
self.n = 10
let xs: Vec<f64> = (0..n).map(|i| i as f64 * 2*PI/n).collect();

factor out f1 and f2 for later reuse